modular_skyrat/modules/xenoarch/code/modules/research/xenoarch/xenoarch_reward.dm

```diff
@@ -0,0 +1,76 @@
+GLOBAL_LIST_INIT(tier1_reward, list(
+	/obj/item/xenoarch/useless_relic = 5,
+	/obj/item/candle = 1,
+	/obj/item/trash/chips = 1,
+	/obj/item/trash/raisins = 1,
+	/obj/item/trash/cheesie = 1,
+	/obj/item/trash/candy = 1,
+))
+
+GLOBAL_LIST_INIT(tier2_reward, list(
+	/obj/item/xenoarch/broken_item/tech = 1,
+	/obj/item/xenoarch/broken_item/plant = 1,
+	/obj/item/xenoarch/useless_relic = 5,
+))
+
+GLOBAL_LIST_INIT(tier3_reward, list(
+	/obj/item/xenoarch/broken_item/weapon = 3,
+	/obj/item/xenoarch/broken_item/illegal = 1,
+	/obj/item/xenoarch/broken_item/alien = 1,
+	/obj/item/stack/spacecash/c10000 = 1,
+))
+
+
+
+GLOBAL_LIST_INIT(tech_reward, list(
+	/obj/item/pipe_dispenser = 1,
+	/obj/item/construction/rcd = 1,
+	/obj/item/anomaly_neutralizer = 1,
+	/obj/item/megaphone = 1,
+	/obj/item/bodybag/bluespace = 1,
+	/obj/item/relic = 1,
+	/obj/item/disk/tech_disk/spaceloot = 1,
+	/obj/item/raw_anomaly_core/random = 1,
+	/obj/item/bag_of_holding_inert = 1,
+	/obj/item/construction/plumbing = 1,
+	/obj/item/mmi/posibrain = 1,
+	/obj/item/storage/portable_chem_mixer = 1,
+))
+
+GLOBAL_LIST_INIT(weapon_reward, list(
+	/obj/item/spear/bonespear = 1,
+	/obj/item/claymore/cutlass = 1,
+	/obj/item/gun/ballistic/automatic/pistol = 1,
```

Comment:
I personally think that if you're gonna go do xenoarch to try and get a free gun as a traitor, maybe it should spawn without ammo? Considering makarov ammo can be made or bought via uplink, or even found in space.

```suggestion
	/obj/item/gun/ballistic/automatic/pistol/no_mag = 1,
```